anymail/backends/base_requests.py

```diff
@@ -20,21 +20,26 @@ def __init__(self, api_url, **kwargs):
         super().__init__(**kwargs)
         self.session = None
 
-    def open(self):
-        if self.session:
-            return False  # already exists
-
+    def create_session(self):
```

Comment:
Please move `create_session()` down with the other overridable methods (after `_send()`), and add a doc comment that it must return a requests.Session and indicating subclasses can override to augment the session, but should generally call the super method to create it.